.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+src/main/resources/log/*
```

didn't notice at first
```suggestion
log/*
```